<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_graph</class>
 <widget class="QMainWindow" name="frm_graph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Вывод графиков</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="bt_averageSpeed">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Средняя скорость</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_distance">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_averageTime">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Среднее время</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_distance">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Пройденное расстояние</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
